src/layout/HeaderSearch.scss

```diff
@@ -0,0 +1,5 @@
+.HeaderSearch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
You may find a good search box implementation at Carbon's official website :) it looks like this:

![image](https://user-images.githubusercontent.com/1916485/102047800-dfcfb200-3e18-11eb-9239-4d0f7ffcc0a0.png)


and it has nice user interactions. 

The implementation is at https://github.com/carbon-design-system/gatsby-theme-carbon/tree/ed2df93443b3aa9230e1162e6759c61fab90cca5/packages/gatsby-theme-carbon/src/components/GlobalSearch